src/wireviz/DataClasses.py

```diff
@@ -119,23 +120,14 @@ def __post_init__(self) -> None:
                 raise Exception('Connectors with style set to simple may only have one pin')
             self.pincount = 1
 
-        if self.pincount is None:
-            if self.pinlabels:
-                self.pincount = len(self.pinlabels)
-            elif self.pins:
-                self.pincount = len(self.pins)
-            else:
+        if not self.pincount:
+            if not self.pins and not self.pinlabels:
                 raise Exception('You need to specify at least one, pincount, pins or pinlabels')
```

Comment:
Are you sure a zero pin connector will crash some part of the program or make absolutely no sense in any use case? If not, then why not allow a zero pin connector, just in case it might become useful in some future use case?

Reply:
I don't see the use for a 0 pin connector at this point; I'd rather wait and see whether it is requested at some point.
Also, there's a difference between specifying 0 pins, and not specifying any pincount, either directly, or via the length of some list.